Automattic/vip-block-data-api
# Bump the PHP version to 8.0 to fix the mu-plugins release process

## Description

This is meant to bump the minimum PHP version to 8.0, to ensure that the mu-plugins release process works.

## Steps to Test

- Run `composer run test` to ensure everything works
- Run `find . -iname '*.php' -exec php -l '{}' \; | grep '^No syntax errors' -v` to ensure nothing is wrong
